week1/Afshin Week1.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # ex1
+
+name = input ("Enter your name :")
+surname = input ("Enter your surname :")
+string_print = f"Your name is {name} and your family is {surname }."
+print (string_print)
+
+
+# ex2
+i = 1
+x = ""
+while i < 6:
+    film = input("Enter name of your favorite film :")
+    x = film + ',' + x + '  '
+    print(x)
+    i += 1
+
```

Comment:
In this exercise you should create a list of movies names, not a string with many movies concatenated